tests/e2e/utils/set-site-verification.js

```diff
@@ -0,0 +1,14 @@
+
+import { wpApiFetch } from './wp-api-fetch';
+
+/**
+ *
```

Comment:
Some proper description and param documentation would be nice 🙂

Reply:
Oops 😄